content/en/user-manual/graphics/shader-chunk-migrations.md

```diff
@@ -29,6 +29,93 @@ By doing this you will no longer see warning messages in the console.
 
 The following tables break down the chunk changes by Engine release.
 
+#### *Engine v1.62*
+
+In 1.62, almost all backend chunks have been changed to accomodate for the more explicit frontend-backend split of the shaders. This means that any custom backend shader chunks must move away from using globals to using the arguments passed to them by the shader backend. 
```

Comment:
We may have to explain what backend/frontend chunks are. We've not shared what this means

Reply:
Good point, done.